step-generation/src/utils/pythonFileUtils.ts

```diff
@@ -121,28 +125,43 @@ export function getLoadAdapters(
       const { parameters, namespace, version } = def
       const adapterSlot = labwareRobotState[id].slot
       const onModule = moduleEntities[adapterSlot] != null
-      const location = onModule
-        ? moduleEntities[adapterSlot].pythonName
-        : PROTOCOL_CONTEXT_NAME
 
-      const adapterArgs = [
-        `${formatPyStr(parameters.loadName)}`,
-        ...(!onModule
-          ? [
-              `location=${formatPyStr(
-                adapterSlot === 'offDeck' ? OFF_DECK : adapterSlot
-              )}`,
-            ]
-          : []),
-        `namespace=${formatPyStr(namespace)}`,
-        `version=${version}`,
-      ].join(',\n')
+      let parentName: string
+      let locationArg: string | undefined
+      if (onModule) {
+        parentName = moduleEntities[adapterSlot].pythonName
+      } else {
+        parentName = PROTOCOL_CONTEXT_NAME
+        locationArg = `location=${formatPyStr(
+          adapterSlot === 'offDeck' ? OFF_DECK : adapterSlot
+        )}`
+      }
```

Comment:
you could default to the else statement, might be a bit cleaner
```suggestion
      let parentName: string = PROTOCOL_CONTEXT_NAME
      let locationArg: string | undefined = `location=${formatPyStr(
          adapterSlot === 'offDeck' ? OFF_DECK : adapterSlot
        )}`
      if (onModule) {
        parentName = moduleEntities[adapterSlot].pythonName
        locationArg = undefined
      } 
```